sonic-xcvrd/xcvrd/xcvrd_utilities/y_cable_helper.py

```diff
@@ -271,37 +323,93 @@ def check_identifier_presence_and_update_mux_table_entry(state_db, port_tbl, y_c
         # Convert list of tuples to a dictionary
         mux_table_dict = dict(fvs)
         if "state" in mux_table_dict:
+
             val = mux_table_dict.get("state", None)
-            if val in ["active", "auto"]:
 
-                y_cable_asic_table = y_cable_tbl.get(asic_index, None)
-                mux_asic_table = mux_tbl.get(asic_index, None)
-                static_mux_asic_table = static_tbl.get(asic_index, None)
-                if y_cable_presence[0] is True and y_cable_asic_table is not None and mux_asic_table is not None and static_mux_asic_table is not None:
-                    # fill in the newly found entry
-                    read_y_cable_and_update_statedb_port_tbl(
-                        logical_port_name, y_cable_tbl[asic_index])
-                    post_port_mux_info_to_db(logical_port_name,  mux_tbl[asic_index])
-                    post_port_mux_static_info_to_db(logical_port_name,  static_tbl[asic_index])
+            if val in ["active", "auto"]:
 
+                # import the module and load the port instance
+                physical_port_list = logical_port_name_to_physical_port_list(
+                    logical_port_name)
+
+                if len(physical_port_list) == 1:
+
+                    physical_port = physical_port_list[0]
+                    if _wrapper_get_presence(physical_port):
+                        port_info_dict = _wrapper_get_transceiver_info(
+                            physical_port)
+                        if port_info_dict is not None:
+                            vendor = port_info_dict['manufacturer'].rstrip()
+                            model = port_info_dict['model'].rstrip()
```

Comment:
Do we just plan to use these strings as-is, or do we plan to do any standardization (e.g., change case, replace spaces with underscores, etc.)?

Reply:
fixed